mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDao.java

```diff
@@ -71,4 +71,5 @@ public interface JobExecutionDao {
    */
   Future<JobExecution> updateBlocking(String jobExecutionId, JobExecutionMutator mutator, String tenantId);
 
+  Future<Boolean> deleteJobExecutionByIds(List<String> ids, String tenantId);
```

Comment:
You are deleting multiple job executions here so better to rename method such as:  deleteJobExecutionsByIds

Reply:
also better to add java docs such as other methods

Reply:
@SerhiiNosko - I have updated the PR as suggested.